tests/test_geoprocessing.py

```diff
@@ -4303,7 +4330,7 @@ def test_convolve_with_byte_kernel(self):
         pygeoprocessing.numpy_array_to_raster(
             array, 255, (20, -20), (0, 0), projection_wkt, signal_path)
         pygeoprocessing.numpy_array_to_raster(kernel.astype(numpy.uint8),
-            255, (20, -20), (0, 0), projection_wkt, int_kernel_path)
+                                              255, (20, -20), (0, 0), projection_wkt, int_kernel_path)

```

Comment:
While the proposed indentation is consistent with PEP8 indentation recommendations, I'm a little surprised that the long-line would be left as-is.  Is this < 80 characters?

Reply:
no, it was longer. more strange stuff from autopep8